denizen_scripts/survival/repo_link/systems/vote_handlers.dsc

```diff
@@ -0,0 +1,151 @@
+vote_reciever:
+  type: world
+  debug: false
+  events:
+    #Vote Recieved:
+      on votifier vote:
+      - define listed_sites <element[to_be_filled_in_when_listed_on_sites]>
+      - define voter <server.match_offline_player[<context.username>]||invalid>
+      - if <[voter]> != invalid:
+        - inject daily_vote_counter
+        - inject weekly_vote_counter
+  #Check for offline keys and award/announce
+      on player joins:
+      - if <player.flag[weekly_crate_pending]>:
+        - narrate: "<&e>You have a <item[weekly_vote_key].display> <&e>pending. Do <&3>/<&b>weeklykeys <&e>to claim it.!"
+      - if <player.flag[enderchest_daily_key_offline]> > 0:
+        - if <player.flag[enderchest_daily_key_offline]> > 1:
+        - narrate "<&e>Your inventory was full and <&a><player.flag[enderchest_daily_key_offline]> <item[daily_vote_key].display>s<&e> were deposited into your enderchest!"
+      - else:
+        - narrate "<&e>Your inventory was full and <&a>a <item[daily_vote_key].display>s<&e> was deposited into your enderchest!"
+      - if <player.flag[inventory_daily_key_offline]> > 0:
+        - if <player.flag[inventory_daily_key_offline]> > 1:
+        - narrate "<&e>You recieved <player.flag[inventory_daily_key]> <item[daily_vote_key].display>s<&e> while offline!"
+      - else:
+        - narrate "<&e>You recieved <&a>a <item[daily_vote_key].display><&e> while you were offline!"
+      - if <player.flag[daily_key_pending]> > 0:
+        - if <player.flag[daily_key_pending]> > 1:
+          - narrate "<&c>You have <player.flag[daily_key_pending]> <item[daily_vote_key].display>s<&c> waiting for you at <&3>/<&b>recoverkeys."
+        - else:
+          - narrate "<&c>You have a <item[daily_vote_key].display><&c> waiting for you at <&3>/<&b>recoverkeys."
```

Comment:
this command doesn't seem to exist?

Reply:
_tableflip_

i thought i wrote it in, ill get that taken care of

Reply:
Added.